src/components/GetInvolved.tsx

```diff
@@ -0,0 +1,22 @@
+import { LINKS } from '@constants';
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+import { Button } from './common';
+
+interface IProps { }
+
+export const GetInvolved: React.FC<IProps> = () => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="get-involved">
+      <p>
+        <Trans i18nKey="GET_INVOLVED.TITLE">
+          BRL-CAD is <span>backed by open</span> source community
+        </Trans>
+      </p>
+      <Button elementType="a" className="brl-button--small" href={LINKS.IRC_LINK} target="_blank">{t('GET_INVOLVED.BUTTON')}</Button>
```

Comment:
How come this line is so long? Do you have the max-length rule disabled in eslint?

Reply:
Fixed

Reply:
Was the rule not added in the eslint file?

Reply:
This rule is usually enabled by default but it has no effect on JSX code, the reason it didn't highlight the error